artifacthub/library/pod-security-policy/forbidden-sysctls/1.0.0/template.yaml

```diff
@@ -51,3 +68,16 @@ spec:
         forbidden_sysctl(sysctl) {
             startswith(sysctl, trim(input.parameters.forbiddenSysctls[_], "*"))
         }
+
+        # * may be used to allow all sysctls
+        allowed_sysctl(sysctl) {
```

Comment:
Is there a difference between this and not using the constraint? Though if we allow globs, this is probably redundant anyway.

Reply:
I think this rule should exist to accept `allowedSysctls: ["*"]`. Although this pattern can be handled by the third rule too, the third rule is for prefix matching.